app/src/main/java/com/example/androidapp/ui/home/HomeFragment.java

```diff
@@ -31,17 +34,25 @@
 import java.util.Random;
 import java.util.Timer;
 import java.util.TimerTask;
+import java.util.ArrayList;
 
 public class HomeFragment extends Fragment {
 
     private FragmentHomeBinding binding;
     private RelativeLayout chic_walking_area;
-    private GifImageView chic;
-    private int x_chic, y_chic;
-    private int next_x_direction, next_y_direction;
+    private GifImageView chic, heart;
```

Comment:
In my environment GifImageView is not found. Are you sure that all the imports are correct? Does it work in yours?